hoomd/data/syncedlist.py

```diff
@@ -192,7 +192,10 @@ def _register_item(self, value):
             return
         else:
             if value._attached:
-                raise RuntimeError(f"Cannot place {value} into two simulations.")
+                if not value._allow_if_updater:
+                    raise RuntimeError(f"Cannot place {value} into list twice.")
+                else:
+                    value._allow_if_updater = False
```

Comment:
I can see that a workaround is needed here because now a `MeshForceCompute` might be added to two synced lists: `integrator.forces` and `updater.forces`. I think we can avoid the workaround by fixing the original error condition. 

The message states "Cannot place {value} into two simulations", but the condition around it is not validating the two simulations part. `value._attach` in `if self._synced` above actually does do that. So it seems that this whole `else` block is not necessary. Try removing it (along with the other `_allow_if_updater` code you added) and see if that works.